cypress/components/menu/menu.cy.js

```diff
@@ -274,6 +274,23 @@ const MenuFullScreenBackgroundScrollTest = () => {
   );
 };
 
+const ClosedMenuFullScreenWithButtons = () => {
```

Comment:
**suggestion** (non-blocking): could you import `ClosedVerticalMenuFullScreenWithButtons` from `vertical-menu-test.stories` instead of creating it again here?

Reply:
totally meant to do that - not sure how I ended up defining it twice and not even using one of them! 🤦 Cheers.

Reply:
Oh hang on, I've only just realised that these are 2 separate components - MenuFullScreen and VerticalMenuFullScreen. So I'm not going to change this, even though the 2 stories are effectively testing the same thing - because they're in 2 separate components.

It is inconsistent that one is defined in a test-stories file and the other directly in the Cypress file, I agree - to be honest I was just going with what the 2 existing Cypress files were doing, and that for (non-vertical) Menu already defines a bunch of 'stories" inside the cypress test file. That probably does want changing but I'd rather leave it to a wider piece of tidying-up than introduce one new story in the test-stories file in a component where we otherwise aren't doing it this way. If QA want this done now they can say so when the ticket reaches them! 😄